newt/target/target.go

```diff
@@ -56,9 +56,13 @@ type Target struct {
 
 func NewTarget(basePkg *pkg.LocalPackage) *Target {
 	target := &Target{
-		TargetY: ycfg.YCfg{},
+		basePkg: basePkg,
 	}
-	target.Init(basePkg)
+
+	if basePkg.SyscfgY.Tree() == nil {
+		panic("ASD")
```

Comment:
While "ASD" seems great, maybe something more descriptive? ;)

Reply:
I really don't see what the problem is.  JK- I don't know how I missed that, thanks!